doc/.static/version_switch.js

```diff
@@ -28,61 +27,57 @@
     versions_dir[ver] = current_version
```

Comment:
@slefrancois I think we should add this line here:
```
current_version = ver
```
This would change `current_version` from `"theano_versions/dev"` to `"dev"`, fixing the name in the version switcher in [L36](https://github.com/Theano/Theano/pull/4604/files#diff-73310b015c0ae02b0ab8561acb3a2a59R36).